src/tip-opengov.ts

```diff
@@ -72,6 +67,36 @@ export async function tipOpenGov(opts: {
           .catch(reject);
       });
   });
+
+  if (tipResult.success && polkassembly) {
+    void (async () => {
+      const condition = async (): Promise<boolean> => {
+        const lastReferendum = await polkassembly.getLastReferendumNumber(track.track.trackNo);
+        return lastReferendum !== undefined && lastReferendum >= referendumId.toNumber();
```

Comment:
this could be vague and there's a chance that someone else right in the same moment posts a referendum to the same track, so we can endup with editing wrong post. 
Small chance but exists
I'm curious if there would be more direct way to get a sense of referendum ID once we created this.